### PR TITLE
Docs: update getting started

### PR DESCRIPTION
**Pull request by larshei**
I just set up my first Ash based JSON API and most of the information I personally needed was available somewhere, but it was split across the getting started guide, the readm, hexdocs and Discord (and in the end I was not sure where I found which information).

I tried to improve the Getting Started guide/tutorial so that it contains all the information I feel I needed to get to a point where I could fire a request against my own API.

All examples are based on the Helpdesk example from Ash's getting started guide.

### CODE REVIEW

File: documentation/tutorials/getting-started-with-json-api.md [hunk 1]
```diff
@@ -1,23 +1,70 @@
...
-## Configure your resources and API
+The resulting JSON APIs follow the specifications from https://jsonapi.org/.
 
-See the DSL documentation for information on configuring
+To add a JSON API, we need to do the following things:
 
-## Create a router
+1. Add the `:ash_json_api` package to your dependencies.
+2. Add the JSON API extension to your `Ash.Resource` and `Ash.Api` modules.
+3. Tell Ash which Resource actions to via the API.
```
Comment (larshei) [inline documentation/tutorials/getting-started-with-json-api.md:12]:
```suggestion
3. Tell Ash which Resource actions to expose via the API.
```

Comment (zachdaniel) [inline documentation/tutorials/getting-started-with-json-api.md:12]:
```suggestion
3. Tell Ash which Resource actions expose over via the API.
```

File: documentation/tutorials/getting-started-with-json-api.md [hunk 2]
```diff
@@ -1,23 +1,70 @@
...
-## Configure your resources and API
+The resulting JSON APIs follow the specifications from https://jsonapi.org/.
 
-See the DSL documentation for information on configuring
+To add a JSON API, we need to do the following things:
 
-## Create a router
+1. Add the `:ash_json_api` package to your dependencies.
+2. Add the JSON API extension to your `Ash.Resource` and `Ash.Api` modules.
+3. Tell Ash which Resource actions expose over via the API.
```
Comment (zachdaniel) [inline documentation/tutorials/getting-started-with-json-api.md:12]:
```suggestion
3. Tell Ash which Resource actions expose over the API.
```

**Comment by zachdaniel** [comment]
🚀 Thank you for your contribution! 🚀

**Review by zachdaniel — APPROVED** [review]
Beautiful!

**Comment by zachdaniel** [comment]
🚀 Thank you for your contribution! 🚀

**Pull request merged**